gyrinx/core/static/core/scss/_classic_card.scss

```
@@ -491,14 +491,34 @@
+    // Two side-by-side columns. Which groups land in which column is decided in
+    // Python (ClassicCard.detail_columns), NOT by CSS multi-column layout:
+    // WebKit doesn't support a multicol container nested inside another
+    // fragmentation context, so when printing (the page being a fragmentation
+    // context) iOS Safari collapsed the whole block to one full-width column.
+    // Two plain flex columns fragment fine and render identically everywhere.
     .cc-detail__cols {
-        column-count: 2;
-        column-gap: var(--cc-detail-gap);
+        display: flex;
+        flex-direction: row;
+        // Columns stretch (the flex default) rather than sizing to their
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)